GitCommands/Git/GitModule.cs

```diff
@@ -2152,8 +2155,7 @@ public async Task<IReadOnlyList<Remote>> GetRemotesAsync()
 
             static IReadOnlyList<Remote> ParseRemotes(ExecutionResult result)
             {
-                IEnumerable<string> lines = result.StandardOutput.LazySplit('\n', StringSplitOptions.RemoveEmptyEntries)
-                    .Concat(result.StandardError.LazySplit('\n', StringSplitOptions.RemoveEmptyEntries));
+                IEnumerable<string> lines = result.StandardOutput.LazySplit('\n', StringSplitOptions.RemoveEmptyEntries);
```

Comment:
I guess the early exit on "not a git repository" below will not be triggered anymore. Should work anyway.

Reply:
Yes, those lines should be removed